css-flexbox-exercise-1/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="pl">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../css/reset.css">
+        <link rel="stylesheet" href="../css/css-flexbox-exercise-1.css">
+        <title>css-flexbox-exercise-1</title>
+    </head>
+    
+    <body>
+        <header>
+           <h1>Lista zakupów</h1>
+        </header>
+        <main>
+            <section class="shopping_list">
+                <div class="box">
+                    <div class="background-img-1">
+                        <h2>Owoce</h2>
+                    </div>
+                    <div class="product-list">
+                        <ul class="ui__ul">
+                            <li>Jabłka</li>
+                            <li>Pomarańcze</li>
+                            <li>Kiwi</li>
+                        </ul>
+                    </div>
+                    <div class="checkbox">
+                        <label>
+                            Owoce kupione
+                            <input type="checkbox" name="terms">
+                        </label>
+                    </div>
+                </div>    
+                <div class="box">
+                    <div class="background-img-2">
+                        <h2>Warzywa</h2>
+                    </div>
+                    <div class="product-list">
+                        <ul class="ui__ul">                
+                            <li>Sałata</li>
+                            <li>Ogórki</li>
+                                <ul class="sub-lista">
+                                    <li>Kiszone</li>
+                                    <li>Świeże</li>
+                                </ul>
+                            <li>Marchew</li>
+                        </ul>
+                    </div>
+                    <div class="checkbox">
+                        <label>
+                            Owoce kupione
+                            <input type="checkbox" name="terms">
+                        </label>
+                    </div>
+                </div>
+                <div class="box">
+                    <div class="background-img-3">
+                        <h2>Nabiał</h2>
+                    </div> 
+                    <div class="product-list">
+                        <ul class="ui__ul">
```

Comment:
nie rozumie troche dlaczego użyleś `ul` w nazwie klasy, czy to ma jakieś powiązanie z tym ze przypisujesz to do tagu `<ul>`? bo na dole masz też masz liste ul która juz ma zupełnie inna klase, dodatkowo w klasie `ui_ul` nie widze zebyś ustawiał style dla unorganized list, chyba tutaj bym sie zastanowiła na inna nazwa, a np klasy `ul` albo `list_unorganized` używałałbym jeżeli by mi reset.css usuwał stylowanie list, i bym musiała dodać od nowa bo wizulanie ul i ol by sie nie różniły

Reply:
Już nie pamiętam, dlaczego takiej nazwy użyłem :P Rzeczywiście sporo niekonsekwencji w moim kodzie. Będę na to zwracał uwagę